src/cov.jl

```diff
@@ -45,6 +60,15 @@ if VERSION < v"0.5.0-dev+679"
     Base.cov(x::DenseMatrix, wv::WeightVec; mean=nothing, vardim::Int=1) =
         scale!(scattermat(x, wv; mean=mean, vardim=vardim), inv(sum(wv)))
 
+
+"""
+    mean_and_cov(x, [wv,] [vardim=1]) -> (Vector, Matrix)
+
+Return the means and covariance matrix as a tuple. A weighting
+vector `wv` can be specified. The function accepts a single
```

Comment:
"The function accepts a single keyword argument" isn't really useful. Go straight to the point. (BTW, the phrasing is a bit different in the previous docstring, maybe decide which one is best and stick with that.)